frontend/src/metabase/css/query_builder.css

```diff
@@ -592,23 +513,6 @@
   min-width: 25em;
 }
 
-.List-item--segment .Icon,
-.List-item--segment .List-item-title {
-  color: var(--filter-color);
-}
-
-.List-item--customfield .Icon,
-.List-item--customfield .List-item-title {
-  color: var(--color-brand);
-}
-
-.FieldList-grouping-trigger {
```

Comment:
Not sure if it's this class, but `<FieldListGroupingTrigger />` renders a bit differently.

#### This PR.
![image](https://user-images.githubusercontent.com/1937582/179954995-80407fa2-e3e0-42a1-bbbb-d1785eec029c.png)

#### master
![image](https://user-images.githubusercontent.com/1937582/179955041-cdac3ad7-fc30-4a00-a5d6-9ccc3b68d3ab.png)

there is a separator in master but it's really subtle, and the subdimension color is also different.

Reply:
Fixed